README.md

```diff
@@ -661,8 +663,27 @@ def test_pebble_exec():
         ctx.on.pebble_ready(container),
         state_in,
     )
+    assert state_out.containers["foo"].get_exec(['ls', '-ll']).stdin == "..."
 ```
 
+Scenario will attempt to find the right `Exec` object by matching the provided
+command prefix against the command used in the ops `container.exec()` call. For
+example if the command is `['ls', '-ll']` then the searching will be:
+
+ 1. an `Exec` with exactly the same as command prefix, `('ls', '-ll')`
+ 2. an `Exec` with the command prefix `('ls', )`
+ 3. an `Exec` with the command prefix `()`
+
+If none of these are found Scenario will raise a `RuntimeError`.
```

Comment:
Any reason why we're not using a custom exception here? Feels like an ExecError would look better than a generic runtime...

Reply:
Fair point. Too much time in the ops world of `RuntimeErrors` everywhere maybe :laughing:. I'll adjust.